lib.nix

```diff
@@ -5,35 +5,41 @@ let
   hostPkgs = import <nixpkgs> {};
   lib = hostPkgs.lib;
 in lib // (rec {
-  # Allows to also generate the key compared to upstream genAttrs
-  genAttrs' = names: fkey: fname:
-    lib.listToAttrs (map (n: lib.nameValuePair (fkey n) (fname n)) names);
-
-  # Given a list of integers and a function,
-  # generate an attribute set with that many nodes and call the function with node index
-  genNodes = nodeLimit: ids: f:
-             genAttrs' (lib.flip builtins.filter ids
-                          (id: id < nodeLimit))
-                       (i: "node${toString i}") f;
+  debugBuild  = let x = (builtins.getEnv "DEBUG_BUILD")  != "";
```

Comment:
What's the difference in size of the binaries when we use dontStrip vs doStrip?

Reply:
I didn't look into that..

Reply:
I understand the idea is to always deploy dontStrip..

Reply:
This doesn't work anyway, we'll add debug flag later on